libs/repositories/src/repos/OrdersAppDataRepository/OrdersAppDataRepository.ts

```diff
@@ -0,0 +1,5 @@
+import { LatestAppDataDocVersion, SupportedChainId } from '@cowprotocol/cow-sdk';
+
+export interface OrdersAppDataRepository {
+  getAppDataForOrders(chainId: SupportedChainId, uids: string[]): Promise<Map<string, LatestAppDataDocVersion>>;
```

Comment:
The appData version can be of any version OR not follow that pattern at all.
So, you could use https://github.com/cowprotocol/app-data/blob/main/src/generatedTypes/index.ts#L36 and maybe also an empty obj because we can't tell.